roles/jenkins_job_launcher/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+# tasks file for jenkins-job-launcher
+    
+- name: trigger jenkins job and wait for it to succeed
```

Comment:
where is the _waiting_ part for the task?

Reply:
The task name was edited, thanks. 
The waiting part is in "Wait for Jenkins status code"